Datahub.Backend/operations/dbOperations.js

```diff
@@ -3,17 +3,31 @@ const AWS = require('aws-sdk')
 
 const dynamo = new AWS.DynamoDB.DocumentClient()
 
+var getTable = function(env) {
+  var table = /live/.test(env) && 'datahub-live-assets' ||
+              /beta/.test(env) && 'datahub-beta-assets' ||
+              'unknown';
```

Comment:
To help us transition between languages better, we should probably try and limit the javascript weirdness (do it in a longer way that reads better) or add a comment explaining what's going on